CMakeLists.txt

```diff
@@ -34,9 +34,9 @@ set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 if(NOT WIN32)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
+    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -O0 -g -DDEBUG")
```

Comment:
I have a question, which compiler you use in appveyor?
I thought it is msvc, according to the content of libuv.in . But -O0, -g and -Os it's not correct options for msvc.
Maybe it should move to proper _if_ , for example:
`if(NOT MSVC`  ?